admin/class-convertkit-admin-refresh-resources.php

```diff
@@ -98,7 +98,7 @@ public function refresh_resources() {
 	public function enqueue_scripts( $hook ) {
 
 		// Bail if we are not on an Edit or Term screen.
-		if ( $hook !== 'edit.php' && $hook !== 'post-new.php' && $hook !== 'term.php' && $hook !== 'post.php' ) {
+		if ( $hook !== 'edit.php' && $hook !== 'post-new.php' && $hook !== 'term.php' && $hook !== 'edit-tags.php' && $hook !== 'post.php' ) {
```

Comment:
Could we have an array of hooks `["edit.php", ... ]` and then use in_array to check it?